microcosm_pubsub/dispatcher.py

```diff
@@ -107,60 +106,24 @@ def find_handler(self, message, bound_handlers):
         """
         try:
             handler = self.sqs_message_handler_registry.find(message.media_type, bound_handlers)
+            # XXX awkward
+            message.handler = handler
             self.opaque["handler"] = titleize(handler.__class__.__name__)
-            return handler
+            return context_logger(
```

Comment:
Something here seems strange. I know it's `microcosm-logging` but it would be nicer to return:
context_logger(self.opaque, handler)`

Reply:
I agree, but I don't want to rewrite microcosm logging today.